Joesdev/dog-messenger
# FASP-30 BE- Fix Form Action Bug on Welcome.blade

## Whats this PR Do?
- Identifies the problem pertaining to the landing page form.  Previously, the form would not submit if an invalid zip code was entered. (such as 99999)
- Fix: Added 'InvalidLocationException', when the form is submitted with an invalid zip code, the page will now refresh with an error which reads 'This Zip Code is Invalid'.

## Steps to Review

1. Navigate to Landing Page Form
2. Enter an Invalid Zip Code (such as 99999 or 11111)
3. Verify an error message is displayed on the page which reads 'This Zip Code is Invalid'.
